### DISCUSSION TITLE
连接微信群、Slack 和 GitHub：社区开放沟通的基础设施搭建 - siwei.io

### DISCUSSION THREAD

**Discussion by giscus** [siwei-io/siwei-io.github.io#21] [Comments]
# 连接微信群、Slack 和 GitHub：社区开放沟通的基础设施搭建 - siwei.io

NebulaGraph 社区如何构建工具让 Slack、WeChat 中宝贵的群聊讨论同步到公共领域

https://www.siwei.io/build-open-communication-infra/

**Comment by wey-gu** [comment]
update:

在圣诞节前，Linen 的工程师允许了 chatroom-syncer 同步到 slack 的消息，详见：

- https://github.com/Linen-dev/linen.dev/issues/761
- https://github.com/wey-gu/chatroom-syncer/pull/42

比如，我们可以在 https://community-chat.nebula-graph.io/c/wechat-sync-venus 看到 wechat 中的群聊的文本备份了！
